packages/docusaurus-theme-classic/package.json

```diff
@@ -14,9 +14,10 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "tsc --noEmit && yarn babel && yarn prettier",
-    "watch": "yarn babel --watch",
-    "babel": "babel src -d lib --extensions \".tsx,.ts\" --ignore \"**/*.d.ts\" --copy-files",
+    "build": "tsc --noEmit && yarn babel:lib && yarn babel:lib-next && yarn prettier",
+    "watch": "yarn babel:lib --watch",
+    "babel:lib": "BABEL_ENV=lib babel src -d lib --extensions \".tsx,.ts\" --ignore \"**/*.d.ts\" --copy-files",
```

Comment:
`cross-env` before setting `BABEL_ENV` should fix the Windows pipeline, same with line below:
 * https://github.com/facebook/docusaurus/blame/master/website/package.json#L13

Reply:
thanks, just forgot about it :)